demisto_sdk/commands/common/update_id_set.py

```diff
@@ -1181,12 +1219,24 @@ def re_create_id_set(id_set_path: Optional[str] = DEFAULT_ID_SET_PATH, pack_to_c
     reports_list = []
     widgets_list = []
     mappers_list = []
+    packs_dict: Dict[str, Dict] = {}
 
     pool = Pool(processes=int(cpu_count() * 1.5))
 
     print_color("Starting the creation of the id_set", LOG_COLORS.GREEN)
 
     with click.progressbar(length=len(objects_to_create), label="Progress of id set creation") as progress_bar:
+
+        if 'Packs' in objects_to_create:
+            print_color("\nStarting iteration over Packs", LOG_COLORS.GREEN)
+            for arr in pool.map(partial(get_pack_metadata_data,
```

Comment:
maybe change `arr` to `pack_data` since this is not an array, also a nice use of partial.

Reply:
done